forte/data/readers/ontonotes_reader.py

```diff
@@ -25,7 +26,80 @@ class OntonotesReader(PackReader):
     download the CoNLL style annotations for the OntoNotes v5.0 release
     – LDC2013T19.tgz obtained from LDC.
 
+    Args:
+        column_format: A list of strings indicating which field each column in a
+            line corresponds to. Available field types include:
+
+            - ``"document_id"``
+            - ``"part_number"``
+            - ``"word"``
+            - ``"pos_tag"``
+            - ``"lemmatised_word"``
+            - ``"framenet_id"``
+            - ``"word_sense"``
+            - ``"speaker"``
+            - ``"entity_label"``
+            - ``"coreference"``
+            - ``"*predicate_labels"``
+
+            Field types marked with ``*`` indicate a variable-column field: it
+            could span multiple fields. Only one such field is allowed in the
+            format specification.
+
+            If a column should be ignored, fill in `None` at the corresponding
+            position.
```

Comment:
I think this function can handle different numbers of fields by providing different lengths of ``column_format``, this might not be immediately clear when reading the docstring.